AmplifyTests/CategoryTests/DataStore/DataStoreCategoryConfigurationTests.swift

```diff
@@ -197,7 +197,7 @@ class DataStoreCategoryConfigurationTests: XCTestCase {
 
         try Amplify.configure(amplifyConfig)
         try Amplify.DataStore.getPlugin(for: "MockSecondDataStoreCategoryPlugin")
-            .save(TestModel.make()) { _ in }
+            .save(TestModel.make(), where: nil) { _ in }
```

Comment:
Seems like th extension exists already, so you could omit the `where: nil` when no condition is needed, right?

Reply:
gave it a try and looks like all the places where we use `Amplify.DataStore.save()` allows us to omit the `where: nil` because we have the `extension DataStoreCategory` which has the default nil value, and calls into the plugin's save. in this case, getPlugin() returns the `DataStoreCategoryPlugin` which is the protocol, and does not have the default `where` argument